python/mxnet/test_utils.py

```diff
@@ -1242,11 +1242,13 @@ def check_speed(sym, location=None, ctx=None, N=20, grad_req=None, typ="whole",
                     exe.arg_dict.items()}
     else:
         assert isinstance(location, dict), "Expect dict, get \"location\"=%s" %str(location)
-        exe = sym.simple_bind(grad_req=grad_req, ctx=ctx,
-                              **{k: v.shape for k, v in location.items()})
+        # exe = sym.simple_bind(grad_req=grad_req, ctx=ctx,
+        #                       **{k: v.shape for k, v in location.items()})
+        exe = sym.bind(ctx=ctx, args=location, grad_req=grad_req)
 
-    for name, iarr in location.items():
-        exe.arg_dict[name][:] = iarr.astype(exe.arg_dict[name].dtype)
+    # for name, iarr in location.items():
```

Comment:
Please remove commented code if not needed.

Reply:
unrelated changes accidentally committed, removed now.